pkg/importer/http-datasource.go

```diff
@@ -196,7 +197,19 @@ func (hs *HTTPDataSource) GetURL() *url.URL {
 
 // GetTerminationMessage returns data to be serialized and used as the termination message of the importer.
 func (hs *HTTPDataSource) GetTerminationMessage() *common.TerminationMessage {
-	return nil
+	if pullMethod, _ := util.ParseEnvVar(common.ImporterPullMethod, false); pullMethod != string(cdiv1.RegistryPullNode) {
```

Comment:
If the func ends with error can it somehow affect the code?

Reply:
The termination message will be `nil`, which is handled by the import controller. It just would lead to the additional information provided by the HTTP datasource getting lost (in this case the labels).